### PR TITLE
Update to 1.17 Pre-Release 2

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
tested in game - this works

### CODE REVIEW

**Comment by DragonEggBedrockBreaking** [comment]
pre-release 2 now

File: gradle.properties [hunk 1]
```diff
@@ -2,15 +2,14 @@
...
 #Fabric api
-	fabric_version=0.32.5+1.16
+	fabric_version=0.34.8+1.17
 
 # Mod Properties
-	mod_version = 1.0.3
+	mod_version = 1.0.4
 	maven_group = net.consistencyteam
 	archives_base_name = obsidian_slab
-
```
Comment (triphora) [inline gradle.properties:16]: what happened here with the line breaks?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Obsidian Slab
 
-Literally just adds the Obsidian slab from Consistency+ as a seprate block.
+Literally just adds the Obsidian slab from Consistency+ as a separate block.
 
-IMS was the guy in charge of alot of the actual code when making the inital block, Siuol took care of the separation after the fact and maintains this specific mod.
+IMS was the guy in charge of a lot of the actual code when making the initial block, Siuol took care of the separation after the fact and maintains this specific mod.
```
Comment (triphora) [inline README.md]: If you're going to fix up some grammar here, don't do a half-arsed job at it; remove the comma splice too :P

**Review by triphora — CHANGES_REQUESTED** [review]

**Comment by Siuolplex** [comment]
You forgot the new tags

File: gradle.properties (hunk 1 above)
Comment (DragonEggBedrockBreaking) [reply gradle.properties:16]: I have no idea. I tried switching from CRLF to LF, but that changed nothing. Frankly, that made it worse, and it stayed like that after i removed the commit...

File: README.md (hunk 2 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: I think I fixed it, assuming I understood correctly what you meant.

**Comment by DragonEggBedrockBreaking** [comment]
> You forgot the new tags

What new tags? Please be more specific.

**Review by triphora — APPROVED** [review]

**Comment by Siuolplex** [comment]
Mineables, mining tiers, etc. I can take em later if you desire

**Comment by DragonEggBedrockBreaking** [comment]
> Mineables, mining tiers, etc. I can take em later if you desire

Yeah, I would rather you fix that, idk what the fuck to do... I just know how to update gradle stuff to jdk 16 and how to use `migrateMappings`...

**Comment by Siuolplex** [comment]
Its datapacks stuff. I'll take it.

**Comment by Siuolplex** [comment]
I'll merge this.

**Pull request merged**